runtime/binding-http/src/main/java/io/aklivity/zilla/runtime/binding/http/internal/config/HttpOptionsConfigAdapter.java

```diff
@@ -396,4 +315,99 @@ public OptionsConfig adaptFromJson(
 
         return httpOptions.build();
     }
+
+    public static void adaptAuthorizationFromObject(
```

Comment:
These addiitonal static methods don't seem like the right abstraction here. Let's discuss the problem you are trying to solve and some different alternatives we can potentially take.

Reply:
My main goal is to reuse this chunk of code in both http and openapi